README.md

```diff
@@ -95,7 +105,7 @@ You can also give it a try using Heroku in one click!
 Be aware that one of the content type builder won't work due to the writing files restriction on the Heroku servers. If you do want to change content types, you need to follow these steps:
```

Comment:
Be aware that the Content Type Builder won't work **due to the restriction of writing files** on the Heroku servers. If you do would like to **change/edit/add Content Types**, you need to follow these steps: